### ISSUE TITLE
Plexiform lesions

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6518]
**Preferred term label:** Plexiform lesions

**Synonyms**
Complex lesions

**Definition (free text, please give PubMed ID)**
Focal proliferation of endothelial cells which form complex capillary-like channels. PMID: 8504682

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary artery morphology HP:0030966

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Suggested term label: Pulmonary arterial plexiform lesion
The term name needs to be comprehensible in isolation

**Comment by pnrobinson** [comment]
added term
